packages/react-core/src/components/LoginPage/__tests__/__snapshots__/LoginFooterItem.test.tsx.snap

```diff
@@ -1,20 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`LoginFooterItem  with custom node 1`] = `
-<div>
-  <div>
-    My custom node
-  </div>
-</div>
-`;
+exports[`LoginFooterItem className is added to the root element 1`] = `"<a target=\\"_blank\\" href=\\"#\\"></a>"`;
```

Comment:
The class isn't being added here, though might be similar to some comments Nicole made about worth noting to readdress in the future.

Reply:
The component doesn't pass it in is why. I'm going to update the component for this since the prop exists, I assume it shouldn't omit the className as it's doing here:
![image](https://user-images.githubusercontent.com/96431149/160005227-9e6f6e7a-8ed3-405f-b821-ecc0701a2d27.png)